### PR TITLE
OSPFV3 model added

### PR DESCRIPTION
**Pull request by rohitthakur2590**

### CODE REVIEW

File: models/iosxr/ospfv3/ospfv3.yaml [hunk 1]
```diff
@@ -0,0 +1,1595 @@
...
+  author: Rohit Thakur (@rohitthakur2590)
+  notes:
+    - Tested against IOS-XR 6.1.3
+    - This module works with connection C(network_cli). See L(the IOS-XR Platform Options,../network/user_guide/platform_iosxr.html)
+  options:
+    config:
+      description: A list of ospfv3 process configuration
+      type: dict
+      suboptions:
+        processes:
```
Comment (rohitthakur2590) [inline models/iosxr/ospfv3/ospfv3.yaml:29]: TODO : add `process_id`

**Comment by samccann** [comment]
Please run the steps from collection_prep here - https://github.com/ansible-network/collection_prep

(I don't know if the scripts will run against a model, but the changes need to happen here).

**Comment by NilashishC** [comment]
Modules that intend to use cli_rm_builder should have model docstrings in the following format: https://github.com/ansible-network/resource_module_models/pull/90/files

**Comment by rohitthakur2590** [comment]
Docstring PR: https://github.com/ansible-network/resource_module_models/pull/95

**Pull request closed without merging**